### ISSUE TITLE
Copy images from previous 

### ISSUE THREAD

**Issue by thibaultmol** [pietervdvn/MapComplete#1403]
I often find myself sometimes adding two car charging poles. Which both fit on the same picture.

So it would be nice if we could reuse the image url instead of uploading the same image to a new url.

So i would assume it would go like 
- add image to object one
- open/create object 2 
- click on "copy images from previous"
(Obv i add specific charging pole id's. So those it'd delete and take specific pictures of the other pole )

**Comment by pietervdvn** [comment]
Yeah, I've had this issue a few times before as well. The 'nearby-images' already does this a little bit, but should have recent/closeby images too.

This might be done in one sweep when we switch to panoramax and redo the entire image upload flow.
